CONTRIBUTING.md

```diff
@@ -64,18 +64,24 @@ in the corresponding go binding repository, [`cdklabs/awscdk-kubectl-go`](https:
     `KUBECTL_VERSION` is v1.20.x, then the `HELM_VERSION` should be v3.8.x.
     - change `SPEC_VERSION` in `.projenrc.js` to reflect the new minor version of kubectl.
     For example, if `KUBECTL_VERSION` is v1.25.0, then `SPEC_VERSION` should be 25.
-    - for an example of code changes done for kubectl v1.22.0, see this [PR](https://github.com/cdklabs/awscdk-asset-kubectl/pull/7).
-5. Run `npx projen` to update the github workflows.
-6. Run `npx projen integ:kubectl-layer:deploy` to ensure that the new versions in the Dockerfile can be successfully downloaded.
-Run `npx projen integ:kubectl-layer:snapshot` if `deploy` succeeds and the snapshot does not get updated.
-7. Run `yarn build` to ensure everything builds correctly.
-8. Commit to your fork and submit a pull request to the repository, _ensuring that you are targeting the correct `kubectl-vY/main` branch_.
-9. A maintainer will review your contribution from there!
-10. ⚠️ IMPORTANT ⚠️ The maintainer should go into the repository settings and update the default branch to the new, latest version.
+    - change the Kubectl Lambda layer class in `src/kubectl-layer.ts` to `KubectlV##Layer`, and its `description` field to reflect the latest versions of Kubectl and Helm being supported.
+    - change `test/kubectl-layer.test.ts` to reflect the new construct's name (changed in the previous step) and that the description
+    verifies the correct versions of Kubectl and Helm.
+    - change `test/kubectl-layer.integ.ts` to reflect the new construct's name (changed in the previous step).
+    - for an example of code changes done for Kubectl v1.22.0, see this [PR](https://github.com/cdklabs/awscdk-asset-kubectl/pull/7).
+5. Run `npx projen compile` to generate the Lambda layer constructs that will be tested.
+6. Run `npx projen` to update the github workflows.
+7. Run `npx projen integ:kubectl-layer:deploy` to ensure that the new versions in the Dockerfile can be successfully downloaded.
+This stage _must succeed_ before proceeding. 
+When it succeeds, confirm that the snapshot in `test/kubectl-layer.integ.snapshot` has been updated. If not, run `npx projen integ:kubectl-layer:snapshot` to update it.
+8. Run `yarn build` to ensure everything builds correctly.
+9. Commit to your fork and submit a pull request to the repository, _ensuring that you are targeting the correct `kubectl-vY/main` branch_.
+10. A maintainer will review your contribution from there!
+11. ⚠️ **IMPORTANT FOR THE MAINTAINER** ⚠️ The maintainer should go into the repository settings and update the default branch to this new, latest version that has just been merged in.
```

Comment:
lets also add the reason why -- github actions only run on default branches and we want the dependencies to be updated in the latest version + previous X versions